### PR TITLE
GitBook links was changed.

### PR DESCRIPTION
**Pull request by Amarouter-Yazilim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,290 +1,2 @@
-# A Byte of Python
```
Comment (swaroopch) [inline README.md:1]: Is this change required? This Markdown file is what people who visit https://github.com/swaroopch/byte-of-python currently see.

Comment (Amarouter-Yazilim) [reply README.md:1]: You're totally right. I was newbie and didn't find out that change. I'll review changes and send the request again.
